index.html

```diff
@@ -1,57 +1,214 @@
+          <section class="header-content">
+        <h1 class="header-title">Hey There. I'm Adaobi</h1>
+        <h2 class="header-subtitle">I’m a Software Developer</h2>
+        <p class="header-text-description">
+          I can help you build a product , feature or website Look through some of my work and experience! If you like
+          what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <ul class="social-media">
+          <li>
+            <a href="#" class="social-media-icon">
+              <img src="./icons/github.png" alt="Github"/>
+            </a>
+          </li>
+          <li>
+            <a href="#" class="social-media-icon">
+              <img src="icons/Linkedin.png" alt="Linkedin"/>
+            </a>
+          </li>
+          <li>
+            <a href="#" class="social-media-icon">
+              <img src="./icons/angel.png" alt="Angel"/>
+            </a>
+          </li>
+          <li>
+            <a href="#" class="social-media-icon">
+              <img src="./icons/twitter.png" alt="Twitter"/>
+            </a>
+          </li>
+          <li>
+            <a href="#"  class="social-media-medium-icon">
+              <img class="image-correction" src="./icons/medium.png" alt="Medium"/>
+            </a>
+          </li>
+        </ul>
+    </section>
+  </header>
+   <main>
+      <section class="new-container">
+        <div class="work-container">
+          <h6 class="recent-work">My Recent Works</h6>
+          <hr class="about-hr">
+        </div>
+        <div class="image-p">
+          <img class="image-placeholder" src="Skeleton/placeholder.png" alt="place holder">
+          <div class="multi">
+           <h6 class="multi-post">Multi-Post Stories</h6>
+           <p class="recent-detail">A daily selection of privately personalized reads; no accounts or sign-ups required.
+            has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+           </p>
+           <ul class="course">
+            <li class="list">css</li>
+            <li class="list">html</li>
+            <li class="list">bootstrap</li>
+            <li class="list">Ruby</li>
+           </ul>
+           <button class="project" type="button">See Project</button>
+          </div>
+        </div>
+        <div class="professional-work">
+          <div class="pro-art">
+           <h6 class="art-print">Profesional Art Printing Data</h6> 
+            <p class="daily-selection">A daily selection of privately personalized reads;
+              no accounts or sign-ups required. has been the industry's standard
+            </p>
+          </div>
+          <ul class="pro-tag">
+            <li class="pro-h">html</li>
+            <li class="pro-h">bootstrap</li>
+            <li class="pro-h">Ruby</li>
+          </ul>
+             <button class="see-proj" type="button">See Project</button>
```

Comment:
- [ ] As already pointed out by the previous reviewer, kindly make sure the size of your button design matches the original design size. In this case, the height is smaller compared to the original design

### Your Design
![image](https://user-images.githubusercontent.com/43995830/159139759-027cf7e6-c57e-4e34-a4f5-13782c162b47.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/159139782-6cde65f8-eeb8-4b4d-bc74-ad4d39edfa69.png)